build/emoji/emoji_gen.mjs

```diff
@@ -372,9 +377,9 @@ endResults.push(goPromise);
 
 // If SERVER_DIR is defined we can update the file emoji_data.go in the server directory
 // eslint-disable-next-line no-process-env
-if (process.env.SERVER_DIR) {
+if (argv['server-env']) {
     // eslint-disable-next-line no-process-env
```

Comment:
`eslint-disable-next-line no-process-env` can be removed since we arent using the env